apps/src/BriskMatch.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include <vector>
+
+#include <opencv2/core.hpp>
+#include <opencv2/features2d.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/imgcodecs.hpp>
+
+static constexpr int NUM_BEST_MATCHES = 30;
+
+int main(int argc, const char* argv[])
+{
+    // vectors for match type and brisk
+    std::vector<cv::String> typeDesc;
+    std::vector<cv::String> typeAlgoMatch;
+    typeDesc.push_back("BRISK");
+
+
+    // create Mat for images
+    auto img1 = cv::imread(argv[1], cv::IMREAD_GRAYSCALE);
+    auto img2 = cv::imread(argv[2], cv::IMREAD_GRAYSCALE);
+
+    // check that images opened correctly
+    if (img1.empty()|| img2.empty() ) {
```

Comment:
Need a space between the first () and the ||. Don't forget to run clang-format before you commit. It'll help catch things like this.